### PR TITLE
Let compression during compaction follow settings for chunk.

### PR DESCRIPTION
**Pull request by baldersheim**
@yngveaasheim or @geirst PR

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java [hunk 1]
```diff
@@ -303,6 +314,7 @@ public void getConfig(ProtonConfig.Summary.Log.Builder log) {
                         if (minFileSizeFactor!=null) log.minfilesizefactor(minFileSizeFactor);
                         if (chunk != null) {
                             chunk.getConfig(log.chunk);
+                            chunk.getConfig(log.compact);
```
Comment (yngveaasheim) [inline config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java:317]: Nitpicking: I guess "chunk" is now more than just "chunk".

File: config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java [hunk 2]
```diff
@@ -281,14 +286,20 @@ public void getConfig(ProtonConfig.Summary.Cache.Builder cache) {
                         }
                     }
 
+                    public void getConfig(ProtonConfig.Summary.Log.Compact.Builder compact) {
+                        if (compression != null) {
+                           compression.getConfig(compact.compression);
```
Comment (yngveaasheim) [inline config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java:291]: Nitpicking: Inconsistent spacing.

File: config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java [hunk 3]
```diff
@@ -245,6 +245,11 @@ public void getConfig(ProtonConfig.Summary.Cache.Compression.Builder compression
                         if (level != null) compression.level(level);
                     }
 
+                    public void getConfig(ProtonConfig.Summary.Log.Compact.Compression.Builder compression) {
+                        if (type != null)  compression.type(ProtonConfig.Summary.Log.Compact.Compression.Type.Enum.valueOf(type.name));
```
Comment (yngveaasheim) [inline config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java:249]: Nitpicking: Spacing is (was already) inconsistent between the 3 getConfig() methods here.

**Review by yngveaasheim — APPROVED** [review]
Some nitpicks; merge at will.

**Review by geirst — APPROVED** [review]
👍

File: config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java (hunk 1 above)
Comment (baldersheim) [reply config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java:317]: Yes, but it is under the chunk tag in services so keeping that link here too.

File: config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java (hunk 3 above)
Comment (baldersheim) [reply config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java]: Unified

File: config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java (hunk 2 above)
Comment (baldersheim) [reply config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java]: Unified

**Pull request merged**